neptune/new/attributes_containers_table.py

```diff
@@ -0,0 +1,233 @@
+        )
+
+
+class AttributesContainersTable:
+    table_entry_cls: type(AttributesContainersTableEntry)
+
+    def __init__(self, backend: NeptuneBackend, entries: List[LeaderboardEntry]):
+        self._backend = backend
+        self._entries = entries
+
+    def to_table_entries(self) -> List[AttributesContainersTableEntry]:
+        return [
+            self.table_entry_cls(
+                backend=self._backend, id_=e.id, attributes=e.attributes
+            )
+            for e in self._entries
+        ]
+
+    def to_pandas(self):
+        # pylint:disable=import-outside-toplevel
+        import pandas as pd
+
+        def make_attribute_value(
+            attribute: AttributeWithProperties,
+        ) -> Optional[Union[str, float, datetime]]:
+            _type = attribute.type
+            _properties = attribute.properties
+            if _type == AttributeType.RUN_STATE:
+                return _properties.value
+            if _type in (
+                AttributeType.FLOAT,
+                AttributeType.INT,
+                AttributeType.BOOL,
+                AttributeType.STRING,
+                AttributeType.DATETIME,
+            ):
+                return _properties.value
+            if (
+                _type == AttributeType.FLOAT_SERIES
+                or _type == AttributeType.STRING_SERIES
+            ):
+                return _properties.last
+            if _type == AttributeType.IMAGE_SERIES:
+                return None
+            if _type == AttributeType.FILE or _type == AttributeType.FILE_SET:
+                return None
+            if _type == AttributeType.STRING_SET:
+                return ",".join(_properties.values)
+            if _type == AttributeType.GIT_REF:
+                return _properties.commit.commitId
+            if _type == AttributeType.NOTEBOOK_REF:
+                return _properties.notebookName
+            if _type == AttributeType.ARTIFACT:
+                return _properties.hash
+            logger.error(
+                "Attribute type %s not supported in this version, yielding None. Recommended client upgrade.",
+                _type,
+            )
+            return None
+
+        def make_row(
+            entry: LeaderboardEntry,
+        ) -> Dict[str, Optional[Union[str, float, datetime]]]:
+            row: Dict[str, Union[str, float, datetime]] = dict()
+            for attr in entry.attributes:
+                value = make_attribute_value(attr)
+                if value is not None:
+                    row[attr.path] = value
+            return row
+
+        def sort_key(attr):
+            domain = attr.split("/")[0]
+            if domain == "sys":
+                return 0, attr
+            if domain == "monitoring":
+                return 2, attr
+            return 1, attr
+
+        rows = dict((n, make_row(entry)) for (n, entry) in enumerate(self._entries))
+
+        df = pd.DataFrame.from_dict(data=rows, orient="index")
```

Comment:
maybe with continuation as:
```python
return pd.DataFrame.from_dict(data=rows, orient="index")
         .reindex(sorted(df.columns, key=sort_key), axis="columns")
```

Reply:
We can't do that since `df` is used used inside `sorted` function